main/modes/mode_slide_whistle.c

```diff
@@ -353,6 +353,12 @@ const noteFrequency_t scl_m_Penta[] =
     C_6, D_SHARP_6, F_6, G_6, A_SHARP_6, C_7,
     C_7, D_SHARP_7, F_7, G_7, A_SHARP_7, C_8,
 };
+const noteFrequency_t scl_Chromatic[] =
```

Comment:
Why does `scl_Chromatic[]` exist twice with different values?

Reply:
Ah, it's a preprocessor thing, see `LOWER_OCTAVE`, which is defined

Reply:
yup!